### PR TITLE
Badges: Support network provider

### PR DESCRIPTION
**Pull request by fabriziovigevani**

### CODE REVIEW

File: src/components/AppBadge/AppBadge.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const AppBadge = React.memo(function AppBadge({
   identifier,
   label,
   labelStyle,
+  networkProvider,
```
Comment (kanaagawa) [inline src/components/AppBadge/AppBadge.js]: The only thing I'm not super convinced is about the naming `provider` here. I would name it `explorer` instead. No sure if we may change in many other places 🤔

Comment (fabriziovigevani) [reply src/components/AppBadge/AppBadge.js]: Yeah maybe explorer makes more sense, just followed the `blockExplorerUrl` convention that uses `provider`. I don't think there's lots of places we need to udpate if we go with explorer

Comment (kanaagawa) [reply src/components/AppBadge/AppBadge.js]: I think now I understand better what they mean by provider. It's the provider that will be used for the explorer like stated here: https://github.com/1Hive/1hive-ui/blob/master/src/utils/web3.js#L186

Regardless is a bit confusing when using the prop `networkProvider` on the component (e.g. AppBadge). Maybe update the props name is enough. What you think?

**Review by 0xGabi — APPROVED** [review]
I tested the changes and include a couple of suggestion to default to blockscout and xDai in #14.

Other than that and what we discuss LGTM!

File: src/components/AppBadge/AppBadge.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/AppBadge/AppBadge.js]: > I think now I understand better what they mean by provider. It's the provider that will be used for the explorer like stated here: https://github.com/1Hive/1hive-ui/blob/master/src/utils/web3.js#L186

Ah yeah that was what i meant by following the convention. I didn't use `provider` as prop for the components as i think in the context of the component `provider` doesn't give enough information about what's providing, but i see that networkProvider actually is not correct. What do you think of using `explorerProvider` for the component prop?

Comment (kanaagawa) [reply src/components/AppBadge/AppBadge.js]: Yes, that's great one!

**Review by 0xGabi — APPROVED** [review]

**Pull request merged**